WcaOnRails/app/helpers/results_helper.rb

```diff
@@ -30,6 +30,9 @@ def historical_pb_markers(results)
   end
 
   def compute_rankings_by_region(rows, continent, country)
+    if rows.empty?
+      return [[]]
```

Comment:
Why are you returning a list within a list here? The original return statement below reads 
```
[rows_to_display, first_continent_index, first_country_index]
```
which is a triple of a list and two ints. You should also return a list and two ints here in my opinion.

Reply:
I see what you mean, but I'm not sure what two ints would make the most sense here, as they simply indicate where in the array the data for continent & country results start. Since there is no such data, perhaps -1?

Reply:
`0, 0` will do. Pretend that the code didn't crash at the `first` invocation, then the other variables would contain an empty list and the `for` loop would effectively not be a loop, leaving the variables at `0`.

Furthermore, I think returning `-1` is not a good idea because it can lead to "IndexOutOfBound" crashes down the line.